assignments/object-functions/code.js

```diff
@@ -0,0 +1,39 @@
+const getX = (object) => object.x
+
+
+const point = (x,y) => {
+  return {'x':x, 'y':y}
+}
+
+const emptyObject = () => {
+  return {}
+}
+
+const distance = (p1, p2) => {
+  return Math.sqrt((((p2.x) - (p1.x))**2) + (((p2.y) - (p1.y))**2))
```

Comment:
This is fine though you can get rid of some of the `()`s. (Though it's of course always fine to put in extra ones to make sure things are grouped correctly.)

```suggestion
  return Math.sqrt((p2.x - p1.x) ** 2 + (p2.y - p1.y) ** 2)
```